packages/components/src/button/style.scss

```diff
@@ -43,6 +43,10 @@
 		outline: 3px solid transparent;
 	}
 
+	&.is-secondary:focus:not(:disabled) {
+		box-shadow: 0 0 0 $black inset, 0 0 0 var(--wp-admin-border-width-focus) $components-color-accent;
+	}
```

Comment:
This style would be better off moving it [here](https://github.com/WordPress/gutenberg/pull/66045/files#diff-0688ad494526b3b921c4444dc8bc6f63efc4ad74c7c1123b061490cfc57f293aR154).